tests/integration/solidity_contracts/Solmate/test_erc721.py

```diff
@@ -14,7 +12,7 @@ async def test_should_set_name_and_symbol(
             deploy_solidity_contract: Callable,
         ):
             erc_721 = await deploy_solidity_contract(
-                "ERC721", "ERC721", "Kakarot NFT", "KKNFT", caller_address=1
+                "Solmate", "ERC721", "Kakarot NFT", "KKNFT", caller_address=1
```

Comment:
why does the test python file live inside the solidity_contracts folder?

Reply:
cc @matthieuauger ; it was decided to ease the testing of .sol files, keeping them close to the test file